readme.html

```diff
@@ -0,0 +1,35 @@
+<!-- File: readme.html
+
+Copyright (c) 2018-2021 Splunk Inc.
```

Comment:
Please update the copyright year to 2021

Reply:
Updated.